mmdeploy/codebase/mmdet/models/detectors/panoptic_two_stage_segmentor.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import copy
+
+import torch
+
+from mmdeploy.core import FUNCTION_REWRITER
+from mmdeploy.utils import is_dynamic_shape
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    'mmdet.models.detectors.panoptic_two_stage_segmentor.'
+    'TwoStagePanopticSegmentor.forward')
+def two_stage_panoptic_segmentor__forward(self,
+                                          batch_inputs,
+                                          data_samples,
+                                          mode='tensor',
+                                          **kwargs):
+    """Rewrite `forward` for default backend. Support configured dynamic/static
+    shape for model input and return detection result as Tensor instead of
+    numpy array.
+
+    Args:
+        batch_inputs (Tensor): Inputs with shape (N, C, H, W).
+        batch_data_samples (List[:obj:`DetDataSample`]): The Data
+            Samples. It usually includes information such as
+            `gt_instance`, `gt_panoptic_seg` and `gt_sem_seg`.
+        rescale (bool): Whether to rescale the results.
+            Defaults to True.
+
+    Returns:
+        tuple[Tensor, Tensor, Tensor, Tensor]:
+            (bboxes, labels, masks, semseg), `bboxes` of shape [N, num_det, 5],
+            `labels` of shape [N, num_det], `masks` of shape [N, roi_H, roi_W],
+            `semseg` of shape [N, num_sem_class, sem_H, sem_W].
+    """
+    ctx = FUNCTION_REWRITER.get_context()
+    data_samples = copy.deepcopy(data_samples)
```

Comment:
Got a RuntimeError: NYI: Named tensors are not supported with the tracer. I used PyTorch 1.12